lib/system/linux/device.c

```diff
@@ -636,3 +636,36 @@ int metal_generic_dev_sys_open(struct metal_device *dev)
 	return 0;
 }
 
+/**
+ * @brief	Read device tree property from sys filesystem.
+ * @param	device		metal_device of the intended
+ *				device tree node
+ * @param[in]	property_name   name of the property to be read
+ * @param[out]	output          output buffer where read data wil be stored 
```

Comment:
indentation

Reply:
Not sure why it is being shown like this in diff, I see indention is proper, when i view file from github.

Reply:
It is due to the mix usage of space and tabl